config/routes.rb

```diff
@@ -12,8 +12,11 @@
   get    "/login",   to: "sessions#new"
   post   "/login",   to: "sessions#create"
   delete "/logout",  to: "sessions#destroy"
+  get 'tasks/index'
+  get 'tasks/:id/success/', to: "tasks#success"
+  get 'tasks/status_run'
   resources :users
   resources :account_activations, only: [:edit]
   resources :password_resets,     only: [:new, :create, :edit, :update]
-  resources :tasks,          only: [:create, :edit, :show, :update, :destroy]
+  resources :tasks,          only: [:create, :edit, :show, :update, :destroy ,:index]
```

Comment:
:newも定義できます
routesの中身が見やすくなりますし、resourcesで定義した方がrailsらしい書き方ですので、
resources :tasks,          only: [:create, :edit, :show, :update, :destroy ,:index, :new]に修正しましょう

また、この修正に伴い、get 'tasks/new' は削除しましょう

Reply:
get'tasks/newを消すと動かなくなるので、入れておきます。indexも同様です。
```suggestion
  resources :tasks,          only: [:create, :edit, :show, :update, :destroy ,:index,:new]
```